.github/workflows/functional_tests.yml

```diff
@@ -0,0 +1,33 @@
+name: Run functional tests using ragger through reusable workflow
```

Comment:
@fbeutin-ledger Can we just add that this worflow also compile & build the Apps. And if we can also rename "functional_tests.yml" by "build_app_and_run_functional_test.yml" (something like that)

Reply:
Done